src/functions.php

```diff
@@ -9,34 +9,6 @@ function connectDB() {
     $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
     return $db;
 }
-
-//----------------------------------------------------------------------------------------------------------------------gets a stat table from the db
-function getTable(string $table,PDO $db): array {
-    $query = $db->prepare("SELECT `{$table}` FROM `{$table}s`");
-    $result = $query->execute();
-    if ($result) {
-        return $query->fetchAll();
-    } else {
-        throw new Exception('error');
-    }
-}
-//--------------------------------------------------------------------------------------------------------------creates the html options for select dropdown
-function dropDownOptions(array $rows): string {
-    $options= '';
-    foreach($rows as $row) {
-        foreach ($row as $key => $value) {
-            $options .= "<option value=\"{$value}\">{$value}</option>";
-        }
-    }
-    return $options;
-}
-//----------------------------------------------------------------------------------------------------------return full html dropdown - select, options, /select
-function createDropdown(string $name) : string {
-    $start = "<select name=\"{$name}\" id=\"{name}\"><option disabled=\"disabled\" selected=\"selected\">$name</option>";
-    $options = dropDownOptions(getTable($name, connectDB()));
-    $end = '</select><br><br>';
-    return $start . $options . $end;
-}
 //---------------------------------------------------------------------------------------------------------creates each sock div and ties together as one string
```

Comment:
again unnecessary separators / comments

The name of the function is `createSockDiv` do you need more explanation?